setup.cfg

```diff
@@ -41,8 +41,6 @@ install_requires =
     jupyter_server>=1.16.0,<2
     notebook_shim>=0.1
     jinja2>=3.0.3
-    ypy-websocket
-    jupyter_ydoc==0.1.10
```

Comment:
Probably this will also require bumping the minimum version of `jupyter_server` if / when it lands there?